setup.cfg

```diff
@@ -6,15 +6,14 @@ author = Fredrik Johansson
 author_email = fredrik.johansson@gmail.com
 url = http://mpmath.org/
 license = BSD
+# When changing the supported python versions, please keep the README, .travis-ci.yml and setup.cfg in sync!
```

Comment:
Maybe it does make sense to add a special section/page (e.g. Development, where can be documented such things as well as release process and so on) for docs, which will have such details?  I don't think it does make sense to do this in the current PR, but you could open a bug and delete all these scattered comments.

Reply:
Yes, as soon as such a document exists then it may be outsourced. Even then, a comment reminding to please read the doc before changing something could be helpful, as it is non-obvious for outsiders how to correctly change something.

In my experience, the CI config is something that is touched by "outsiders" more often than expected; often something suddenly breaks due to external dependencies and then someone randomly notices it and proposes some random fix without considering all the details. Just like me :-D

If whoever decides on merging this PR wants to remove the comment I'm happy to remove it.

Reply:
> comment reminding to please read the doc before changing something could be helpful

I strongly disagree.

> as it is non-obvious for outsiders how to correctly change something

Most non-obvious things/workflows - probably should be documented in the project docs.  Contributors must
read one.  It may worth to point on this in the [github-specific CONTRIBUTING file](https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/setting-guidelines-for-repository-contributors).

> If whoever decides on merging this PR wants to remove the comment I'm happy to remove it.

It's difficult to keep similar comments in sync.  Better - have some project-wide guidelines in a single place.

Reply:
I removed these for now, I care most about getting it merged in so I can have my other pull requests merged in so I can then switch back my code to an upstream mpmath version.